content/concepts/anomaly_scoring.md

```diff
@@ -19,7 +19,7 @@ Individual rules designed to detect specific types of attacks and malicious beha
 
 Once all of the rules that inspect *request* data have been executed, *blocking evaluation* takes place. If the anomaly score is greater than or equal to the inbound anomaly score threshold then the transaction is *denied*. Transactions that are not denied continue on their journey.
 
-![Diagram showing an example where the inbound anomaly score threshold is set to 5. A first example request accumulates an anomaly score of 2 and is allowed to pass at the blocking evaluation step. A second example request accumulates an anomaly score of 7 and is denied at the blocking evaluation step.](anomaly_scoring/as_inbound_no_fonts.svg?height=36em)
+![Diagram showing an example where the inbound anomaly score threshold is set to 5. A first example request accumulates an anomaly score of 2 and is allowed to pass at the blocking evaluation step. A second example request accumulates an anomaly score of 7 and is denied at the blocking evaluation step.](as_inbound.svg)
```

Comment:
When I look at the current file on GitHub, the image is displayed correctly. With your change it is not.

Reply:
Where? when I look at https://coreruleset.org/docs/concepts/anomaly_scoring/ or my local hugo instance, the image is missing.

Reply:
Hm, that's annoying. It's rendered correctly on GitHub: https://github.com/coreruleset/documentation/blob/main/content/concepts/anomaly_scoring.md.